tests/dpsdgym/README.md

```diff
@@ -56,17 +56,17 @@ Make sure you have the following installed in a conda env, before running the ev
 ## Usage
 Once the configuration/datasets are set up, the entire evaluation pipeline, with all metrics, can be run through the eval script
 ```
-python main.py all
+python main.py
 ```
-Flags can be used to specify which metrics to run in the evaluation. You could equivalently run
+Flags can be specified to specify which metrics, epsilon and datasets should be used to run the evaluation:
 ```
-python main.py wasserstein pmse ml_eval sra
+python main.py -d bank adult -e 0.01 0.1 1 -m pmse ml_eval wasserstein

```

Comment:
Part of the reason we avoided this approach is that mlflow did not properly parse the arguments for the remote scenario.

Can you run these changes with mlflow run "tests/dpsdgym" -P {param}={val} to verify the changes are compatible?

Reply:
I am going to assume this comment is not a blocker since those configuration points were not there to begin with